packages/graph-explorer/src/connector/useGEFetchTypes.ts

```diff
@@ -82,6 +82,8 @@ export type NeighborsRequest = {
   vertexId: VertexId;
   /**
    * Source vertex type.
+   *
+   * Used only by the SPARQL implementation to patch the edges returned with the source vertex type.
    */
   vertexType: string;
```

Comment:
optional?

Reply:
Unfortunately, it is required by 1 out of the 3 code paths. Throwing an exception would make for a worse developer experience, in my opinion, since it moves the failure to runtime (and only if you test that code path).